corehq/apps/reports/templates/reports/standard/base_template.html

```diff
@@ -18,6 +24,13 @@
 {% endblock %}
 
 {% block js-inline %}{{ block.super }}
+    <script>
+        $(document).ready(function () {
```

Comment:
Should be `$(function() {` for jQuery 3. I think this style will still work, but it's deprecated: https://jquery.com/upgrade-guide/3.0/#deprecated-document-ready-handlers-other-than-jquery-function